api-spec/protobuf/tdexa/v1/analytics.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+
+package tdexa.v1;
+
+/**
+ * Analytics service spec
+ */
+service Analytics {
+  // returns all markets and its balances in time series
+  rpc MarketsBalances(MarketsBalancesRequest) returns (MarketsBalancesReply) {}
+  // returns all markets and its prices in time series
+  rpc MarketsPrices(MarketsPricesRequest) returns (MarketsPricesReply) {}
+}
+
+message MarketsBalancesRequest {
+  // from_time fetch balances from time in past
+  string from_time = 1;
+  // fetch balances for specific market, if not provided all market's balances will be returned
+  int32 market_id = 2;
```

Comment:
very likely clients would want to ask for many markets in batch at the same time, and receive alll in bulk

Reply:
should we leave option to fetch one? or we should always return all?

Reply:
can we have a `repeated` field? so client can call one or many as they wish